src/main/scala/io/movies/model/Movie.scala

```diff
@@ -0,0 +1,5 @@
+package io.movies.model
+
+case class Movie(title: String, director: String, releaseDate: Short)
+
+case class RegisteredMovie(id: Short, title: String, director: String, releaseDate: Short)
```

Comment:
The `serial` data type 4-bytes integer in [PostgreSQL](https://www.postgresql.org/docs/9.5/datatype-numeric.html), so we should use `Int` instead of `Short` for the `id` field.

Reply:
See https://github.com/gutiory/MoviesServer/pull/1/commits/13385b63cc666e9b55340b7f95f59b8397dc3ea1